docs/source/tutorial_swap_components.md

```diff
@@ -0,0 +1,60 @@
+# Swapping Out Model Subcomponents
```

Comment:
say Transformer explicitly, since this only works for Transformers

Reply:
It technically can be added to any model (or any class at all, really). But I'll take your advice to make it easier to think about for now.